debug_toolbar/templates/debug_toolbar/base.html

```diff
@@ -31,7 +31,7 @@
     </ul>
   </div>
   <div class="djdt-hidden" id="djDebugToolbarHandle">
-    <div title="{% translate 'Show toolbar' %}" id="djShowToolBarButton">
+    <div title="{% translate 'Show toolbar' %}" id="djShowToolBarButton" class="{% if toolbar.health_level %} djdt-toolbarhandle-health-{{ toolbar.health_level }}{% endif %}">
```

Comment:
I generally do not like uppercase letters much in ID and class attributes, but I wonder if we should keep capitalization for internal consistency here? Any thoughts?

Reply:
They are inconsistent right now. Some places use camelCase when starting class names with `dj`, while others use lowercase with dashes (mostly when starting with `djdt`). 

Rightnow, I can change them to camelcase if you want, but for consistency the changes in whole project will be required in future.

Reply:
Ah, don't worry about it then. Being as consistent as possible is good enough.